src/verter/store/postgres.clj

```diff
@@ -60,6 +60,21 @@
                         {:return-keys true :builder-fn jdbcr/as-unqualified-lower-maps})
          (mapv (partial v/from-row opts)))))
 
+(defn- find-facts-for-multiple-ids
```

Comment:
>[!NOTE]
_not just here, but everywhere in the PR_

for "`find-facts-for-multiple-ids`"

"`ids`" already means "multiple ids"

so "`find-facts-for-multiple-ids`" should be "`find-facts-for-ids`"